config/channels/in-memory-channel/README.md

```diff
@@ -0,0 +1,63 @@
+# In-Memory Channels
+
+In-memory channels are a best effort Channel. They should **NOT** be used in
+Production. They are useful for development.
+
+They differ from most Channels in that they have:
+
+- No persistence.
+  - If a Pod goes down, messages go with it.
+- No ordering guarantee.
+  - There is nothing enforcing an ordering, so two messages that arrive at the
+    same time may go downstream in any order.
+  - Different downstream subscribers may see different orders.
+- No redelivery attempts.
+  - If downstream rejects a request, a log message is written, but that request
+    is never sent again.
+
+### Deployment steps:
+
+1. Setup [Knative Eventing](../../../DEVELOPMENT.md).
+1. Apply the 'in-memory-channel' CRD, Controller, and Dispatcher.
+   ```shell
+   ko apply -f config/channels/in-memory-channel/in-memory-channel.yaml
+   ```
+1. Create InMemoryChannels
+
+   ```yaml
+   apiVersion: messaging.knative.dev/v1alpha1
+   kind: InMemoryChannel
+   metadata:
+     name: foo
+   spec:
+   ```
+
+### Components
+
+The major components are:
+
+- InMemoryChannel Controller
+- Channel Controller
+- Channel Dispatcher
+- Channel Dispatcher Config Map.
+
+The ClusterChannelProvisioner Controller and the Channel Controller are
+colocated in one Pod.
+
+```shell
+kubectl get deployment -n knative-eventing in-memory-channel-controller
+```
+
+The Channel Dispatcher receives and distributes all events. There is a single
+Dispatcher for all in-memory Channels.
+
+```shell
+kubectl get deployment -n knative-eventing in-memory-channel-dispatcher
+```
+
+The Channel Dispatcher Config Map is used to send information about Channels and
```

Comment:
The current HEAD `in-memory` CCP no longer uses the ConfigMap, does this new version?

Reply:
nope, cruft.